src/tempdir.rs

```diff
@@ -5,13 +5,14 @@
 // SPDX-License-Identifier: (Apache-2.0 AND BSD-3-Clause)
 
 //! Structure for handling temporary directories.
-
-use libc;
+use std::env::temp_dir;
 use std::ffi::{CString, OsStr, OsString};
 use std::fs;
 use std::os::unix::ffi::OsStringExt;
 use std::path::{Path, PathBuf};
 
+use libc;
```

Comment:
nit: why not have this where it previously was?

Reply:
I structured the `use`s in the following groups: `std` usages in one group, other libs usages in another, same crate modules usages respectively in another.